Makefile

```diff
@@ -437,8 +437,10 @@ check-doc: doc
 	@find . -name "*.md" | xargs git diff --exit-code -- \
 	|| (echo "Please update generated documentation by running 'make doc' and committing the changes" && false)
 
-check-doc-links:
-	cd ./tools/doc-validator && go run . ../../docs/sources/
+# Tool is developed in the grafana/technical-documentation repository:
+# https://github.com/grafana/technical-documentation/tree/main/tools/doc-validator
+check-doc-validator: ## Check documentation using doc-validator tool
+	docker run -v "$(CURDIR)/docs/sources:/docs/sources" grafana/doc-validator:latest ./docs/sources
```

Comment:
Should we always run `docker pull grafana/doc-validator:latest` to ensure we always run on latest?

Reply:
Good idea, added in 034cec264